src/plugin/ss_plugin.rs

```diff
@@ -22,6 +22,13 @@ pub fn plugin_cmd(plugin: &PluginConfig, remote: &ServerAddr, local: &SocketAddr
 
     if let Some(ref opt) = plugin.plugin_opt {
         cmd.env("SS_PLUGIN_OPTIONS", opt);
+        #[cfg(target_os = "android")]
+        {
+            // Add VPN flags to the commandline as well
+            if opt.contains(";V") {
+                cmd.arg("-V");
+            }
+        }
```

Comment:
In shadowsocks-libev:

```c
#ifdef __ANDROID__
    extern int vpn;
    if (vpn)
        cork_exec_add_param(exec, "-V");
#endif
```

I think plugin should read this flag from `SS_PLUGIN_OPTIONS` instead of `-V`, since many programs uses `-V` for printing version.

Reply:
It's a workaround for v2ray-plugin, it currently only accepts command line args. Maybe we can apply it on v2ray-plugin only to avoid other potential issues.